lib/livebook_web/live/home_live/import_file_upload_component.ex

```diff
@@ -0,0 +1,80 @@
+defmodule LivebookWeb.HomeLive.ImportFileUploadComponent do
+  use LivebookWeb, :live_component
+
+  @impl true
+  def mount(socket) do
+    {:ok,
+     socket
+     |> assign(:error, false)
+     |> allow_upload(:notebook, accept: ~w(.livemd), max_entries: 1)}
+  end
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div class="flex-col space-y-5">
+      <p class="text-gray-700">
+        You can drag and drop a file to this modal at any time to open it up as a notebook.
+      </p>
+      <form id="upload-file-form"
+        phx-submit="save"
+        phx-change="validate"
+        phx-drop-target={@uploads.notebook.ref}
+        phx-target={@myself}
+        phx-hook="DragAndDrop"
+        class="flex flex-col items-start"
+      >
+        <div style="display: none">
+          <%= live_file_input @uploads.notebook %>
+        </div>
+        <div name="dropzone" class="flex flex-col justify-center items-center w-full rounded-xl border-2 border-dashed border-gray-400 h-48">
+          <%= if @uploads.notebook.entries == [] do %>
+            <span name="placeholder" class="font-medium text-gray-400">Drop your notebook here</span>
+          <% else %>
+            <%= for file <- @uploads.notebook.entries do %>
+              <div class="flex items-center">
+                <span class="font-medium text-gray-400"><%= file.client_name %></span>
+                <button type="button" class="icon-button" phx-click="clear-file" phx-target={@myself} data-toggle tabindex="-1">
```

Comment:
```suggestion
                <button type="button" class="icon-button" phx-click="clear-file" phx-target={@myself} tabindex="-1">
```

(unless I'm missing the use of it)